dev/index.js

```diff
@@ -4,9 +4,35 @@ import Boilerplate from './Boilerplate'
 import Buoy from 'buoy'
 import VueRouter from 'vue-router'
 import router from './router'
+import VeeValidate from 'vee-validate'
+
+const validationConfig = {
```

Comment:
Does this match the validationConfig that Marty is using?

Reply:
Not sure.  This is the one copied directly from your branch.

Reply:
then yes, yes it is.

Reply:
👍